src/bin/grin.rs

```diff
@@ -198,6 +198,13 @@ fn real_main() -> i32 {
 				.short("l")
 				.long("port")
 				.help("Port on which to run the wallet listener")
+				.takes_value(true))
+			.arg(Arg::with_name("method")
+				.short("m")
+				.long("method")
+				.help("Which method to use for communication")
+				.possible_values(&["http", "keybase"])
+				.default_value("http")
```

Comment:
I believe there is something wrong here. Arg `method` is already defined at line 241.